README.md

```diff
@@ -30,11 +30,19 @@ Import ChatJS before the AWS SDK, and ensure the AWS SDK version you are using c
 #### A note for Scenarios 2 and 4.
 When using the SDK and ChatJS, you may remove the SDK from ChatJS to ensure lack of import conflicts. However, this should not be relevant if the order in which you are importing these libraries is the order reflected above.
 
-### Downloading ChatJS
+### Using ChatJS from npm
+
+`npm install amazon-connect-chatjs -save`
```

Comment:
double `--` for `save`? also the doc is no longer showing `--save` as an option https://docs.npmjs.com/cli/install

btw https://stackoverflow.com/questions/36022926/what-do-the-save-flags-do-with-npm-install#comment79204893_36023156 says `--save` is the default behavior now.

Reply:
Got it, I'll get rid of it